### PR TITLE
Change some FIXMEs to TODOs

### PR DESCRIPTION
**Pull request by axic**
We use more TODOs more often and actually these cases were not bugs, just missing features.

### CODE REVIEW

File: lib/fizzy/execute.cpp [hunk 1]
```diff
@@ -565,7 +565,8 @@ std::unique_ptr<Instance> instantiate(Module module,
 
     // We need to create instance before filling table,
     // because table functions will capture the pointer to instance.
-    // FIXME: clang-tidy warns about potential memory leak for moving memory (which is in fact
+    //
+    // NOTE: clang-tidy warns about potential memory leak for moving memory (which is in fact
```
Comment (gumb0) [inline lib/fizzy/execute.cpp:569]: @chfast wanted to deal with it at some point, that's why it was `FIXME`

Comment (axic) [reply lib/fizzy/execute.cpp:569]: So change it to `TODO`? I don't think it is a bug in our code.

Comment (gumb0) [reply lib/fizzy/execute.cpp:569]: Yes, I'd make it `TODO`

Comment (axic) [reply lib/fizzy/execute.cpp]: Fixed.

**Pull request merged**